src/app/app.component.ts

```diff
@@ -31,7 +34,20 @@ export class AppComponent {
   isLoggedIn$: Observable<String | null> = this.usersService.isLoggedIn$
   user$: Observable<User | null> = this.usersService.user$
 
-  constructor(private usersService: UsersService) { }
+  localStorage: Storage | undefined
+
+  isServer: boolean | null = null
+
+  constructor(private usersService: UsersService, public translate: TranslateService, @Inject(DOCUMENT) private document: Document, @Inject(PLATFORM_ID) platformId: Object) {
+    this.isServer = isPlatformServer(platformId)
+    this.localStorage = document.defaultView?.localStorage
+    translate.addLangs(['en', 'es', 'ru', 'lv'])
```

Comment:
Ah great you already have something here, good! You should only have this in one place since you'll fetch the language settings and anything related to translations on the first webpage load. I would suggest moving it to a service though for readability and proper structure